hermes-api/src/main/java/pl/allegro/tech/hermes/api/SubscriptionHealthProblem.java

```diff
@@ -0,0 +1,116 @@
+package pl.allegro.tech.hermes.api;
+
+public enum SubscriptionHealthProblem {
```

Comment:
OK, so Map<HealthProblem, HealthProblemCalculator> in HealthProblemChecker, filled by Spring DI? This decouples HealthProblems fromHealthProblemCalculators and vice versa.

I didn't really get the fragment about configuration parameters :P